Sources/Fakery/Generators/Cat.swift

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
@51d4 Could u please remove the copyright?

Reply:
Good catch! Thanks :)